Date.js

```diff
@@ -0,0 +1,45 @@
+function convertEpochToDateTime(epoch){
+	return new Date(epoch);
+}
+
+function convertDateTimeToEpoch(dateTime){
+	return dateTime.getTime();
+}
+
+function getSundayOfCurrentWeekInEpoch(todaysDateInDateTime){
+	var dayOfWeek = todaysDateInDateTime.getDay();
+	return new Date(todaysDateInDateTime.getTime() - dayOfWeek*86400000).setHours(0,0,0,0);
```

Comment:
minor point but we should use spaces before and after operators like this:
`+	return new Date(todaysDateInDateTime.getTime() - dayOfWeek * 86400000).setHours(0,0,0,0);`